components/megamenu/MegaMenuLayout/mega-menu-layout.tsx

```diff
@@ -32,10 +32,11 @@ const MegaMenuLayout: React.FC<MegaMenuWrapperProps> = ({
               passHref
               className="unstyled flex items-center gap-1 whitespace-nowrap"
             >
-              <div className="flex items-center justify-center">
+              {/* eslint-disable-next-line tailwindcss/no-arbitrary-value */}
+              <div className="flex min-w-[4rem] max-w-[14rem] items-center justify-center">
```

Comment:
Assuming this is planned tech debt (I know there are lots of refactors coming to this menu)

Reply:
yeah just trying to make it layout nicely